src/views/AdminSettings.vue

```diff
@@ -0,0 +1,105 @@
+<!--
+  - @copyright Copyright (c) 2022 Nikita Toponen <natoponen@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<SettingsSection :title="t('notifications', 'Notifications defaults')">
+		<p class="settings-hint">
+			{{ t('notifications', 'Configure the default notification settings for new users') }}
+		</p>
+
```

Comment:
This is the old way, the new way would be:
https://nextcloud-vue-components.netlify.app/#/Components/NcSettings?id=ncsettingssection

```suggestion
	<SettingsSection :title="t('notifications', 'Notifications defaults')"
		:description="t('notifications', 'Configure the default notification settings for new users')">
```